code/modules/mob/living/simple_animal/friendly/dog.dm

```diff
@@ -552,6 +552,7 @@ GLOBAL_LIST_INIT(strippable_corgi_items, create_strippable_list(list(
 	playsound(src, 'sound/magic/demon_dies.ogg', 75, TRUE)
 	var/mob/living/simple_animal/pet/dog/corgi/narsie/N = new(loc)
```

Comment:
A reason to test stuff I guess with a second commit; You're using a single letter variable, could you make it a bit more descriptive while you're here?

Reply:
I am not using a single letter variable, it was already there, i just added the mind transfer line. But i can change the variable name, no problem.

Reply:
you mind doin that?